groovy/postInit/chemistry/inorganic_chemistry/elements/d_block/group9/CobaltChain.groovy

```diff
@@ -2,19 +2,77 @@ import globals.Globals
 import static globals.CarbonGlobals.*
 
 ROASTER = recipemap('roaster')
+EBF = recipemap('electric_blast_furnace')
 PBF_RECIPES = recipemap("primitive_blast_furnace")
+FLUIDIZEDBR = recipemap('fluidized_bed_reactor')
+BR = recipemap('batch_reactor')
+ELECTROLYTIC_CELL = recipemap('electrolytic_cell')
+
+//ROAST RECOVERY OF As2O3
+//COBALTITE MODIFICATION
+mods.gregtech.electric_blast_furnace.removeByInput(120, [metaitem('dustCobaltite')], [fluid('oxygen') * 3000])
+
+
+EBF.recipeBuilder()
+        .inputs(ore('dustCobaltite'))
+        .fluidInputs(fluid('oxygen') * 9000)
```

Comment:
2CoAsS + 9/2 O2 -> 2CoO + As2O3 + 2SO2, you need to fix the cobaltite input and CoO output.

Reply:
since Cobaltite is raw ore, I'm going to leave the input alone, so I only fixed the CoO Output

Reply:
that's not how it works :skull: